mslib/msui/msui_mainwindow.py

```diff
@@ -917,16 +1027,20 @@ def create_view(self, _type, model):
         view_window = None
         if _type == "topview":
             # Top view.
-            view_window = topview.MSUITopViewWindow(mainwindow=self, model=model,
+            view_window = topview.MSUITopViewWindow(parent=self, mainwindow=self, model=model,
```

Comment:
ToDo: This looks like a regression https://github.com/Open-MSS/MSS/issues/2007

Reply:
adding parent here blocks the subwidget showing up in the taskbar for windows

![image](https://github.com/user-attachments/assets/141b10f1-760a-4d5b-a47c-36197ac6ff2d)

Reply:
https://github.com/Open-MSS/MSS/pull/2554

Reply:
The dependent widgets of a dockingwidget are not shown because they have the same parent from the main widget used.

![widgets](https://github.com/user-attachments/assets/c69fa49b-9f1b-489a-ba83-fb6954e5987c)